src/Integration.Vsix.UnitTests/Settings/GeneralOptionsDialogPageTests.cs

```diff
@@ -130,7 +130,7 @@ public void ClickHyperlink_ShowWikiCommandIsCalled()
             page.Control.ShowWikiHyperLink.Command.Execute(null);
 
             // Assert
-            browserService.Verify(x => x.Navigate("https://github.com/SonarSource/sonarlint-visualstudio/wiki"), Times.Once);
+            browserService.Verify(x => x.Navigate("https://docs.sonarsource.com/sonarlint/visual-studio/team-features/connected-mode/"), Times.Once);
```

Comment:
This link might change as I had to ask kevin for a more detailed sub link. It is mentioned in our Options window with the text "Note: per-user settings are ignored for C/C++ for solutions using connected mode. See the  *wiki link* for more information." so if there is nothing more precise from kevin I think linking to the connected-mode page makes most sense.